document_page_tag/tests/test_document_page_tag.py

```diff
@@ -14,9 +14,21 @@ def test_document_page_tag(self):
             testtag,
             self.env["document.page.tag"].name_create("Test"),
         )
+
+        # check multiple record creation
+        testtag1 = self.env["document.page.tag"].create(
+            [{"name": "test1"}, {"name": "test"}, {"name": "test2"}]
+        )
+        self.assertEqual(len(testtag1), 2)
+        self.assertEqual(testtag1[0].name, "test1")
+        self.assertEqual(testtag1[1].name, "test2")
+        testtag2 = self.env["document.page.tag"].create([{"name": "test"}])
```

Comment:
separate from test1

Reply:
Done

Reply:
Changed variable name to snakecase format

Reply:
Missed  commit precommit changes after pushing, updated the required changes